patches/server/0936-Uncompressed-Regions.patch

```diff
@@ -0,0 +1,36 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: Peter Crawley <peterzcrawley@gmail.com>
+Date: Tue, 25 Oct 2022 17:38:53 +0100
+Subject: [PATCH] Uncompressed Regions
+
+
+diff --git a/src/main/java/net/minecraft/world/level/chunk/storage/RegionFile.java b/src/main/java/net/minecraft/world/level/chunk/storage/RegionFile.java
+index fc3442b4c7e1f22080fe6bf36d4fade162d6709e..0de66062218f8b83111e7a2e10eb6925089c86a0 100644
+--- a/src/main/java/net/minecraft/world/level/chunk/storage/RegionFile.java
++++ b/src/main/java/net/minecraft/world/level/chunk/storage/RegionFile.java
+@@ -425,11 +425,11 @@ public class RegionFile implements AutoCloseable {
+     // Paper end
+ 
+     public RegionFile(Path file, Path directory, boolean dsync) throws IOException {
+-        this(file, directory, RegionFileVersion.VERSION_DEFLATE, dsync);
++        this(file, directory, RegionFileVersion.ACTIVE_VERSION, dsync); // Paper - Uncompressed Regions
+     }
+     // Paper start - add can recalc flag
+     public RegionFile(Path file, Path directory, boolean dsync, boolean canRecalcHeader) throws IOException {
+-        this(file, directory, RegionFileVersion.VERSION_DEFLATE, dsync, canRecalcHeader);
++        this(file, directory, RegionFileVersion.ACTIVE_VERSION, dsync, canRecalcHeader); // Paper - Uncompressed Regions
+     }
+     // Paper end - add can recalc flag
+ 
+diff --git a/src/main/java/net/minecraft/world/level/chunk/storage/RegionFileVersion.java b/src/main/java/net/minecraft/world/level/chunk/storage/RegionFileVersion.java
+index 97d4ae5619dcc0922e0381b1bb45a135f514e3af..62f4592b346f2ac216cb8d7ab191c7177c831da0 100644
+--- a/src/main/java/net/minecraft/world/level/chunk/storage/RegionFileVersion.java
++++ b/src/main/java/net/minecraft/world/level/chunk/storage/RegionFileVersion.java
+@@ -30,6 +30,7 @@ public class RegionFileVersion {
+     }, (outputStream) -> {
+         return outputStream;
+     }));
++    public static final RegionFileVersion ACTIVE_VERSION = io.papermc.paper.configuration.GlobalConfiguration.get().chunkLoading.saveRegionsUncompressed ? VERSION_NONE : VERSION_DEFLATE; // Paper - Uncompressed Regions
```

Comment:
Make sure that the GlobalConfiguration is loaded before this static field is initialized. Otherwise you'll want to use a Supplier here. 

Actually, its probably better to make this a Supplier regardless, just to make sure no class load order issues arise.